server/controllers/accounts.js

```diff
@@ -11,6 +11,27 @@ const AccountsController = {
       .then(() => res.status(200).json(newAccount))
       .catch((error) => res.status(400).json('Error: ' + error));
   },
+
+  authenticate(req, res) {
+    Account.findOne(
+      { username: req.body.username },
+      function (error, foundUser) {
+        if (!error) {
+          if (foundUser) {
+            if (foundUser.password == req.body.password) {
+              res.json(foundUser).send;
+            } else {
+              res.send('Incorrect password');
+            }
+          } else {
+            res.send('You have not been registered yet');
+          }
+        } else {
+          res.send(error);
```

Comment:
We're wondering if a switch statement would be easier to read than three nested if statements?